portal/views/teacher/teach.py

```diff
@@ -89,18 +91,17 @@ def teacher_classes(request):
         if form.is_valid():
             created_class = create_class(form, teacher)
             messages.success(request, "The class '{className}' has been created successfully."
-                             .format(className = created_class.name))
-        return HttpResponseRedirect(reverse_lazy('teacher_class', kwargs={ 'access_code': created_class.access_code }))
+                             .format(className=created_class.name))
+        return HttpResponseRedirect(reverse_lazy('teacher_class', kwargs={'access_code': created_class.access_code}))
```

Comment:
Majority of changes in this file are to stop codeclimate bitching.

Actual code changes at `341`, `540`.